test/Sema/access-level-import-flag-check.swift

```diff
@@ -28,3 +31,8 @@ package import PackageLib // expected-error@:1 {{Access level on imports require
 internal import InternalLib // expected-error@:1 {{Access level on imports require '-enable-experimental-feature AccessLevelOnImport'}}
 fileprivate import FileprivateLib // expected-error@:1 {{Access level on imports require '-enable-experimental-feature AccessLevelOnImport'}}
 private import PrivateLib // expected-error@:1 {{Access level on imports require '-enable-experimental-feature AccessLevelOnImport'}}
+
+//--- Client.swiftinterface
+// swift-interface-format-version: 1.0
+// swift-module-flags: -enable-library-evolution
+public import Swift
```

Comment:
Should we also test that other access levels are accepted, even though we do not plan to print them yet? I'm thinking of `package` in particular, which we do probably want to print in the future in some interfaces.

Reply:
Good idea, I've added them to the test.